spec/services/hubee_api_client_spec.rb

```diff
@@ -0,0 +1,65 @@
+RSpec.describe HubEEAPIClient do
+  let(:hubee_api_client) { described_class.new }
+
+  before do
+    stub_request(:post, hubee_api_client.send(:hubee_auth_url))
+      .to_return(status: 200, headers: { 'Content-Type' => 'application/json' }, body: { "access_token" => "token" }.to_json)
+  end
+
+  describe '#access_token' do
+    subject(:access_token) { hubee_api_client.access_token }
+
+    context 'when the API returns a 200' do
+      before do
+        stub_request(:post, hubee_api_client.send(:hubee_auth_url))
+          .to_return(status: 200, headers: { 'Content-Type' => 'application/json' }, body: { "access_token" => "token" }.to_json)
+      end
+
```

Comment:
```suggestion
      before do
        stub_request(:post, hubee_api_client.send(:hubee_auth_url))
          .to_return(status: 200, headers: { 'Content-Type' => 'application/json' }, body: { "access_token" => "token" }.to_json)
      end

```

C'est déjà fait ligne 4